Source/WebKit/NetworkProcess/NetworkDataTask.h

```diff
@@ -150,6 +150,9 @@ class NetworkDataTask : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<N
 
     virtual void setTimingAllowFailedFlag() { }
 
+    inline size_t bytesTransferredOverNetworkDelta();
+    size_t totalBytesTransferredOverNetwork() { return m_totalBytesTransferredOverNetwork; }
```

Comment:
should be `const`

Reply:
It changes the member variable so it cannot be `const`.

Reply:
I'll change the name to `calculateBytesTransferredOverNetworkDelta` to make not look like getter.

Reply:
Are you commenting on the wrong line? The line I commented on looks like a trivial getter than CAN be const.